airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -1147,6 +1152,57 @@ public void testCancelSyncWhenCancelledWhenWorkerIsNotRunning() throws Exception
     assertEquals(JobStatus.CANCELLED, resp.get().getJob().getStatus());
   }
 
+  @Test
+  @Order(22)
+  @DisabledIfEnvironmentVariable(named = "KUBE",
```

Comment:
Is there a reason to not run on Kube?

Reply:
originally i thought we would have to do some extra stuff to get it working on kube, but i guess that's not the case. it's been removed.